rasa/shared/utils/llm.py

```diff
@@ -208,3 +209,22 @@ def embedder_factory(
         return embeddings_cls(**parameters)
     else:
         raise ValueError(f"Unsupported embeddings type '{typ}'")
+
+
+def get_prompt_template(
+    default_prompt_template: Text, jinja_file_path: Optional[Text]
```

Comment:
STY: I'd suggest changing the parameter order, I am used to defaults being the last / later argument.